api/models/unit.js

```diff
@@ -0,0 +1,65 @@
+const mongoose = require('mongoose');
+const Schema = mongoose.Schema;
+const companySchema = require('./company');
+
+/**
+ * Schema Notes:
+ * 
+ * Unit: The Unit resource is a generic term for a rentable space.
+ *      kind: (Required) The kind of unit available. Only the following options should be allowed: "seat", "desk", "small office", "large office", "floor".
+ *      floor: (Required) An integer which represents the floor the rentable unit is on.
+ *      special_monthly_offer: An optional integer field. If filled in, represents a special deal that the company will be paying in cents. For example, if the unit is being leased for $600/mo, this field would be 600000.
+ *      company: The Company resource, refer to schema in ./api/models/company.js
+ *      
+ */
+
+ /**
+  * Schema
+  * Note: I originally built out the company and employee schemas to seperate model files, but out of frustration with bugs i encountered i opted to build out the full schema in this unit.js file once more. 
+  * I also opted out of frustration/rush to build all the routes out in the units.js file rather than separate routes. I have however uploaded the additional model files
+*/
+
+const schema = new Schema({
+    unit: {
```

Comment:
I think you probably don't need this key. If you consider this schema is the Unit, having another `unit` key inside of it is a bit confusing. It leads to things like what you have below where you say `unit.unit...`.